src/modules/Book.js

```diff
@@ -1,7 +1,7 @@
 class Book {
   constructor(title) {
     this.title = title;
-    this.isbn = Date.now().toString(); // Generate a unique identifier for each book
+    this.isbn = Date.now().toString(); // Generate a unique identifier for each book and manager it
     this.completed = false;
   }
```

Comment:
- [ ] I am selecting this portion of code because I cannot find the clear completed functionality. When I clear the button to clear completed tasks I get the image below. The task is cleared but kindly get rid of the error message. I have to refresh the page to see the remaining tasks.

  ![image](https://github.com/tchasinga/To-do-list-Microverse/assets/36542986/9ebca91d-1f9f-4e13-9674-3a8203c79cb1)